Configuration/Form/Checkboxlink.yaml

```diff
@@ -0,0 +1,37 @@
+TYPO3:
+  CMS:
+    Form:
+      prototypes:
+        standard:
+          formEditor:
+            dynamicRequireJsModules:
+              additionalViewModelModules:
+                - 'TYPO3/CMS/FormTools/Backend/FormEditor/CheckboxlinkViewmodel'
```

Comment:
ViewModel. 'm' uppercase as you can see in line above

Reply:
Done